lib/vagrant-service-manager/plugin_util.rb

```diff
@@ -94,6 +94,15 @@ def self.execute_and_exit_on_fail(machine, ui, command)
         ui.error e.message.squeeze
       end
 
```

Comment:
I find PluginUtil a bit odd. It starts getting very long and seems to be a class for everything. I think there are more than one class which are needed there.